src/backend/parser/cypher_clause.c

```diff
@@ -2443,19 +2647,15 @@ static transform_entity *transform_VLE_edge_entity(cypher_parsestate *cpstate,
     TargetEntry *te = NULL;
     RangeFunction *rf = NULL;
     RangeTblEntry *rte = NULL;
-    FuncCall *func = NULL;
     Alias *alias = NULL;
     Node *var = NULL;
     transform_entity *vle_entity = NULL;
 
     /* it better be a function call node */
     Assert(IsA(rel->varlen, FuncCall));
 
-    /* get the function */
-    func = (FuncCall*)rel->varlen;
```

Comment:
Needs to be removed. The updated VLE patch will take care of this

Reply:
Sorry, I deleted it. When merging, inserted it due to a compilation warning.